magenta-lib/src/main/scala/magenta/artifact/ArtifactHelper.scala

```diff
@@ -0,0 +1,30 @@
+package magenta.artifact
+
+import com.amazonaws.services.s3.AmazonS3
+import com.amazonaws.services.s3.model.{ListObjectsV2Request, S3ObjectSummary}
+
+import scala.collection.JavaConverters._
+
+object ArtifactHelper {
+  def listObjects(path: S3Location)(implicit s3Client: AmazonS3) = {
```

Comment:
Ha - totally hadn't noticed that. Mostly only touched magenta.

Would you be happy for that to be extracted into the magenta library if I can make it work?